content/system-firmware.md

```diff
@@ -21,7 +21,7 @@ These instructions are for System76 owners who have been prompted for a firmware
 ### Before You Begin
 
 * Make sure the laptop is **plugged into the AC Adapter** for the entire firmware updating process. USB Type-C charging is **not** supported during firmware updates on the following models: Galago Pro (galp3-c, galp4), Darter Pro (darp5, darp6)
-* **Disconnect external devices**, including any displays or USB devices.
+* **Disconnect external devices**, including any displays or USB devices. The firmware update will restart and the update will not be written if external devices are connected (no error is shown).
```

Comment:
I don't think every single display or USB device will necessarily have this effect. Might need testing, but the wording probably needs to at least say "may" instead of "will."

Reply:
Thanks for the comment. I updated to text.